auth-tokens-filter/src/main/java/com/palantir/tokens/auth2/http/BearerTokenLoggingFilter.java

```diff
@@ -79,6 +79,6 @@ private void setUnverifiedContext(ContainerRequestContext requestContext, String
     }
 
     public static String getRequestPropertyKey(String key) {
-        return "com.palantir.tokens.auth." + key;
+        return "com.palantir.tokens.auth2." + key;
```

Comment:
I was in doubt, so I checked what was done in http-remoting: https://github.com/palantir/http-remoting/commit/689b315eebda9183a89ef84895fa20e133e9f484#diff-73656c4e1289f899124f962ba82c494f